src/netaddress.h

```diff
@@ -462,11 +462,31 @@ class CSubNet
         bool SanityCheck() const;
 
     public:
+        /**
+         * Construct an invalid subnet (empty, `Match()` always returns false).
+         */
         CSubNet();
+
+        /**
+         * Construct from a given network start and number of bits (CIDR mask).
+         * @param[in] addr Network start. If not IPv4 or IPv6, then a signle-host subnet is created.
+         * @param[in] mask CIDR mask, must be in [0, 32] for IPv4 addresses and in [0, 128] for IPv6
+         * addresses. Otherwise an invalid subnet is created. Ignored for non-IP `addr`.
+         */
         CSubNet(const CNetAddr& addr, uint8_t mask);
+
+        /**
+         * Construct from a given network start and mask.
+         * @param[in] addr Network start. If not IPv4 or IPv6, then a signle-host subnet is created.
+         * @param[in] mask Network mask, must be of the same type as `addr` and not contain 0-bits
+         * followed by 1-bits. Otherwise an invalid subnet is created. Ignored for non-IP `addr`.
+         */
```

Comment:
readability suggestion
```diff
         /**
          * Construct from a given network start and number of bits (CIDR mask).
-         * @param[in] addr Network start. If not IPv4 or IPv6, then a signle-host subnet is created.
-         * @param[in] mask CIDR mask, must be in [0, 32] for IPv4 addresses and in [0, 128] for IPv6
-         * addresses. Otherwise an invalid subnet is created. Ignored for non-IP `addr`.
+         * @param[in] addr  Network start. If not IPv4 or IPv6, then a single-host subnet is created.
+         * @param[in] mask  CIDR mask, must be in [0, 32] for IPv4 addresses and in [0, 128] for IPv6
+         *                  addresses. Otherwise an invalid subnet is created. Ignored for non-IP `addr`.
          */
         CSubNet(const CNetAddr& addr, uint8_t mask);
 
         /**
          * Construct from a given network start and mask.
-         * @param[in] addr Network start. If not IPv4 or IPv6, then a signle-host subnet is created.
-         * @param[in] mask Network mask, must be of the same type as `addr` and not contain 0-bits
-         * followed by 1-bits. Otherwise an invalid subnet is created. Ignored for non-IP `addr`.
+         * @param[in] addr  Network start. If not IPv4 or IPv6, then a single-host subnet is created.
+         * @param[in] mask  Network mask, must be of the same type as `addr` and not contain 0-bits
+         *                  followed by 1-bits. Otherwise an invalid subnet is created. Ignored for non-IP `addr`.
          */
         CSubNet(const CNetAddr& addr, const CNetAddr& mask);
```

Reply:
This 3-column-layout has a maintainability problem when a parameter with longer name is added or existent one renamed.

Reply:
Yes, though I think it's worth it. Anyway, just a suggestion.